src/run.js

```diff
@@ -36,13 +36,14 @@ const minimalcss = async options => {
       if (/data:image\//.test(request.url)) {
         // don't need to download those
         request.abort()
-      } else if (/\.(png|jpg|jpeg$)/.test(request.url)) {
+      } else if (/\.(png|jpg|jpeg|webp|gif$)/.test(request.url)) {
         request.abort()
       } else if (stylesheetAstObjects[request.url]) {
         // no point downloading this again
         request.abort()
+      } else if (options.skippable && options.skippable(request)) {
```

Comment:
Excellent! I think this is a great start. But it's currently undocumented and that's almost more important than the functionality. 

Can you update the README about this being a possible options key and add an example (perhaps one with your mapbox URLs). 
In fact, under the "API" section of the README you can just mention it's a callable that gets a Request object and expects to return a boolean if defined. 

Then add another section where you describe an example of a more elaborate version. Perhaps an example that uses those mapbox.com URLs. Also, it would be cool if the example demonstrates how you can also skip the URL based on the Content-Type header.

Reply:
but this is request, not response. I suppose it does not have Content-Type header.

Reply:
You're so totally right. Sorry, I rushed that comment. Ignore that bit about `content-type`.
However, your documentation should point to Request on this URL: https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-request